DeskReserve/Exceptions/EntityNotFoundException.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Runtime.Serialization;
+
+namespace DeskReserve.Exception
+{
+    [Serializable]
```

Comment:
Why exception is serialzable?

Reply:
Serializing exceptions can be beneficial when you have errors occurring in different components of your system. If these components are distributed across different locations, serialization can help easily share error information between them.   For example, if an error is thrown in the repository and you want to pass it back to the controller through your service, serialization can facilitate this process.